app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -266,9 +288,11 @@ fun Topic(
                     .clickable {
                         toggleBookmark = !toggleBookmark
                         onClickBookmark(toggleBookmark)
-                    }, painter = painterResource(
+                    },
+                painter = painterResource(
                     id = if (toggleBookmark) R.drawable.ic_star_fill else R.drawable.ic_star_empty
-                ), tint = if (toggleBookmark) MainColor01 else Gray04, contentDescription = null
+                ),
+                tint = if (toggleBookmark) MainColor01 else Gray04, contentDescription = null
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)